sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentModelAdministrationClient/DocumentModelAdministrationClientLiveTests.cs

```diff
@@ -236,8 +238,7 @@ public async Task AdminOps(bool useTokenCredential)
 
             await client.DeleteModelAsync(modelId);
 
-            RequestFailedException ex = Assert.ThrowsAsync<RequestFailedException>(() => client.GetModelAsync(modelId));
-            Console.WriteLine(ex.ErrorCode);
+            Assert.ThrowsAsync<RequestFailedException>(() => client.GetModelAsync(modelId));
```

Comment:
Not sure why we were printing the error code in this test (debugging?).

Reply:
Not needed for our SDK testing, but at some point we wanted to verify the error codes the service returns as they kept changing them.
They have been stable for a while so we can keep deleting printing/verifications